.github/workflows/check_lint_build_release.yaml

```diff
@@ -125,7 +125,7 @@ jobs:
         with:
           # Stable, as of January 2015. If updating, also update
```

Comment:
Should update the comment

Reply:
What do you think of this? https://github.com/LayerTwo-Labs/bip300301_enforcer/pull/220

Lets us simplify down to only one place we specify the Rust version. Also has the benefit of getting rustup users (AKA everyone?) to automatically be on the correct version